clang/include/clang/CIR/Dialect/IR/CIRCUDAAttrs.td

```diff
@@ -35,4 +35,21 @@ def CUDAKernelNameAttr : CIR_Attr<"CUDAKernelName",
   let assemblyFormat = "`<` $kernel_name `>`";
 }
 
+def CUDABinaryHandleAttr : CIR_Attr<"CUDABinaryHandle",
+                                  "cuda_binary_handle"> {
```

Comment:
We should follow the convention elsewhere: `cu.binary_handle` (in a later PR, you can change the other one from `cuda_kernel_name` to `cu.kernel_name`).